src/components/alert/index.tsx

```diff
@@ -45,9 +45,22 @@ const Alert: FC<AlertProps> = ({
         <AlertDescription>{description}</AlertDescription>
         {link ? (
           link.as === 'button' ? (
-            <Link onClick={() => link.onClick?.()}>{link.text}</Link>
+            <Link
+              as="button"
+              onClick={() => link.onClick?.()}
+              textAlign="left"
+              display="inline-block"
+              width="fit-content"
```

Comment:
I am wondering if it's possible to do it only with `width=max-content` then we can remove the display property?

Reply:
Yes! It works! Thank you 👑